core/hash/transform_keys_spec.rb

```diff
@@ -84,7 +84,7 @@
     end
   end
 
-  ruby_version_is "2.5.1" do
+  ruby_version_is "2.5.1"..."3.0" do
```

Comment:
I think it should be `3.0.1` (also below) since 3.0.0 behaves likes 2.5.1 and the result is `{:b=>1, :c=>2}`.
I'll push a commit to do that.

Reply:
Actually it's even 3.0.2 it seems.

Reply:
Right! Thank you for your additional fix.